### PR TITLE
fixed proclib detection logic

### PR DESCRIPTION
**Pull request by adn1107**
This fixes a bug where the pproc output on some systems are different than standard ADCD image, causing the zos release can't be derived correctly. The bug was discovered by Josh of the Broker team late last year when he tested the IMS Operator on the VSI environment.

### CODE REVIEW

File: playbooks/roles/ims_common/tasks/main.yml [hunk 1]
```diff
@@ -71,6 +71,9 @@
           - active_ctl_msg is defined 
           - '"EMERGENCY BACKUP OR OVERRIDE" in active_ctl_msg | join'
 
+      - wait_for:
+          timeout: 10
```
Comment (jashnani) [inline playbooks/roles/ims_common/tasks/main.yml:75]: Should we add a comment explaining why timeout is needed?

File: playbooks/roles/ims_common/tasks/main.yml [hunk 2]
```diff
@@ -88,6 +91,9 @@
           - display_result.msg is defined 
           - '"Success" not in display_result.msg | join'
 
+      - wait_for:
+          timeout: 10
```
Comment (jashnani) [inline playbooks/roles/ims_common/tasks/main.yml:95]: Ditto.

File: playbooks/test.yml [hunk 3]
```diff
@@ -21,18 +21,97 @@
 
   tasks:
   - block:
+    # - name: copy java program to zos
```
Comment (jashnani) [inline playbooks/test.yml:24]: Should this test playbook be added to `build_ignore` in galaxy.yml so it doesn't get built into the collection? 

Interesting - I just checked the previous release and this file existed before also but didn't get added to the built collection. Do you know how this is being ignored?

**Review by jashnani — APPROVED** [review]
LGTM. Couple minor comments below:

File: playbooks/test.yml (hunk 3 above)
Comment (adn1107) [reply playbooks/test.yml:24]: maybe ignored by CICD?

File: galaxy.yml [hunk 4]
```diff
@@ -73,20 +73,10 @@ homepage: https://www.ibm.com/products/z-and-cloud-modernization-stack
...
-  - .release-it.yml 
-  - .travis.yml 
-  - CHANGELOG.md 
-  - build
-  - Jenkinsfile
-  - ansible-lint-config.yml
-  - Dockerfile 
-  - .whitesource
+  - packages
+  - local
```
Comment (jashnani) [inline galaxy.yml:81]: Is `local` folder used somewhere?

File: playbooks/roles/ims_common/tasks/main.yml (hunk 1 above)
Comment (adn1107) [reply playbooks/roles/ims_common/tasks/main.yml:75]: ok, will add comment in the next commit

File: galaxy.yml (hunk 4 above)
Comment (adn1107) [reply galaxy.yml:81]: created by Operator SDK

File: playbooks/test.yml (hunk 3 above)
Comment (jashnani) [reply playbooks/test.yml:24]: It'd be good for us to confirm. I also noticed that some other files/playbooks are also being ignored, perhaps intentionally (for ex: playbooks/delete-ims-datasets.yml)

**Pull request merged**